scoring_engine/checks/wordpress.py

```diff
@@ -0,0 +1,11 @@
+from scoring_engine.engine.basic_check import BasicCheck
+
+
+class WordpressCheck(BasicCheck):
+    required_properties = ['useragent', 'vhost', 'username', 'password', 'uri']
+    CMD = 'curl -s -S -4 -v -L --cookie-jar - --header {0} -A {1} --data log={2}&pwd={3} {4}'
+
+    def command_format(self, properties):
+        host_header = 'Host: ' + properties['vhost']
+        host_uri = self.host + ':' + str(self.port) + properties['uri']
+        return (host_header, properties['useragent'], host_uri)
```

Comment:
The reason is because you're only returning 3 entries in the list here, they must match the same number of placeholders as https://github.com/scoringengine/scoringengine/pull/598/files#diff-c2d34569c9b76da11d27020be61f8ce9R6

You'll want to do something like https://github.com/scoringengine/scoringengine/blob/master/scoring_engine/checks/mysql.py#L9 to get the account username and password.